premailer/premailer.py

```diff
@@ -111,7 +114,8 @@ def __init__(self, html, base_url=None,
                  base_path=None,
                  disable_basic_attributes=None,
                  disable_validation=False,
-                 cache_css_parsing=True):
+                 cache_css_parsing=True,
+                 add_leftover_css=True):
```

Comment:
Can you reverse the meaning of this parameter. It's generally a good idea to have "negative" defaults. E.g. instead you can make it `disable_leftover_css=False`.